nessie/merged/sis_profile.py

```diff
@@ -164,3 +165,15 @@ def merge_sis_profile_plans(academic_status, sis_profile):
         # Add plan unless it's a duplicate.
         if not next((p for p in sis_profile['plans'] if p.get('description') == plan_feed.get('description')), None):
             sis_profile['plans'].append(plan_feed)
+
+
+def merge_sis_profile_withdrawal_cancel(academic_status, sis_profile):
+    print(academic_status.get('currentRegistration', {}))
```

Comment:
@paulkerschen  Is the `print()` above wanted?

Reply:
@johncrossman Nope! Amending.